### PR TITLE
feat(wallet): disable multi wallets (Mobile)

### PR DESCRIPTION
**Pull request by omniwired**

### CODE REVIEW

**Comment by omniwired** [comment]
https://github.com/TERITORI/teritori-dapp/pull/1079#issuecomment-2019223426

**Review by n0izn0iz — DISMISSED** [review]
Code LGTM

File: packages/screens/Mini/Profile/ProfileScreen.tsx [hunk 1]
```diff
@@ -88,6 +88,11 @@ export const ProfileScreen: ScreenFC<"MiniProfile"> = ({ navigation }) => {
     });
   };
 
+  // disable multiple wallet creation, remove to re enable multiple
```
Comment (hthieu1110) [inline packages/screens/Mini/Profile/ProfileScreen.tsx:91]: This comment is clear enough to me if I read this code here  but in some case if someone debug and follow the code, he can see that var is const then believe that its value will be immutable but not in this case.

so maybe we should create 2 consts :

const profileScreen = [<settings>, <dapps>] 
const profileForMultiWalletsScreen = [<add account>, ... profileScreen];

File: packages/screens/Wallet/Screens/NativeWallet.tsx [hunk 2]
```diff
@@ -43,33 +45,47 @@ const NativeWallet: ScreenFC<"NativeWallet"> = () => {
...
+          }
           {nativeWallet && (
             <CustomButton
               title="Continue"
               style={{
                 marginVertical: layout.spacing_x2,
               }}
               onPress={(_, navigation) => {
-                navigation.navigate("MiniTabs", { screen: "MiniChats" });
+                if (appMode === "mini") {
```
Comment (hthieu1110) [inline packages/screens/Wallet/Screens/NativeWallet.tsx:66]: should we create enum AppMode instead of string ?

Comment (omniwired) [reply packages/screens/Wallet/Screens/NativeWallet.tsx:66]: an Enum doesn't make sense. The comparison is always against === "mini".
What I would make sense is to refactor this code everywhere and replace it with something like:
`useIsMiniMode()`

**Comment by WaDadidou** [comment]
@hthieu1110 @n0izn0iz @omniwired 
What is the status of this PR ?
Is the issue still relevant ?

**Pull request closed without merging**